### PR TITLE
Remote signaller buff

### PR DESCRIPTION
**Pull request by starch70**
## About the PR
remote signaller now has range of 40m
advanced has range of 500000m
c4 bundle now has signaller and network configurator
ougghh advanced signaller in uplink 8 tc
also fixed universal access config not having ntrep and bso

## Why / Balance
my stolen seismic charges and c4 will hold every substation hostage for i am RETEP and i am EVIL 

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**
:cl:
- add: c4 bundle gets remote signaller and network configurator now
- add: advanced signaller in uplink 8 tc
- tweak: remote signaller range is now 40m and advanced one is practically infinite range now
- fix: universal access configurator has NTrep and BSO in it now (this means nothing to not-shitmins/mappers)

## Summary by CodeRabbit

- **New Features**
	- Introduced new duffel bag bundles with specific items, including the `ClothingBackpackChameleonFill` and `ClothingBackpackDuffelSyndicateC4tBundle`.
	- Added an "Advanced Remote Signaller" to the uplink catalog, enhancing disruption tools available to players.
	- Expanded access levels for the `AccessConfiguratorUniversal`, now including `NanotrasenRepresentative` and `BlueshieldOfficer`.

- **Improvements**
	- Enhanced operational range for signaller devices, with the `RemoteSignaller` now effective up to 40 meters and the `RemoteSignallerAdvanced` having no range limit.

- **Content Adjustments**
	- Updated contents of various duffel bags, refining item specifications and categories, including surgical tools and themed items.

### CODE REVIEW

**Comment by a-person5660** [comment]
holy based

**Comment by Vlax2nd** [comment]
corn starch here with another fire PR

File: Resources/Prototypes/Entities/Objects/Devices/Electronics/signaller.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   parent: BaseItem
   id: RemoteSignaller
   name: remote signaller
-  description: A handheld device used for remotely sending signals to objects within a small radius of about 15 meters.
+  description: A handheld device used for remotely sending signals to objects within a small radius of about 250 meters. # goobstation
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Objects/Devices/Electronics/signaller.yml:7]: way too large of a buff

Comment (Piras314) [reply Resources/Prototypes/Entities/Objects/Devices/Electronics/signaller.yml:7]: also that's not a small radius lol

File: Resources/Prototypes/Entities/Objects/Devices/Electronics/signaller.yml [hunk 2]
```diff
@@ -40,6 +40,6 @@
       right:
       - state: advanced-inhand-right
   - type: WirelessNetworkConnection
-    range: 50
+    range: 500000 # goobstation
```
[the remark this answers is not in the document]
Comment (Piras314) [reply Resources/Prototypes/Entities/Objects/Devices/Electronics/signaller.yml:43]: :true:

File: Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml [hunk 3]
```diff
@@ -193,6 +193,8 @@
       contents:
         - id: C4
           amount: 8
+        - id: RemoteSignallerAdvanced # goobstation
```
Comment (Piras314) [inline Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml:196]: I'd say you should put a regular signaller with C4 bundle (I agree with buffing the regular signaller, not to the crazy extent you have but still) and add the advanced signaller to the uplink as its own item.

**Review by Piras314 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Devices/Electronics/signaller.yml (hunk 1 above)
Comment (a-person5660) [reply Resources/Prototypes/Entities/Objects/Devices/Electronics/signaller.yml:7]: imo if it gets any buff then it should be to 30-50 meters, 250 is waay to much for a roundstart item.

File: Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml (hunk 3 above)
Comment (starch70) [reply Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml:196]: no its fine trust the process

File: Resources/Prototypes/Entities/Objects/Devices/Electronics/signaller.yml [hunk 4]
```diff
@@ -4,7 +4,7 @@
   parent: BaseItem
   id: RemoteSignaller
   name: remote signaller
-  description: A handheld device used for remotely sending signals to objects within a small radius of about 15 meters.
+  description: A handheld device used for remotely sending signals to objects within a small radius of about 40 meters. # goobstation
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Objects/Devices/Electronics/signaller.yml:7]: remove small

File: Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml (hunk 3 above)
Comment (Piras314) [reply Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml:196]: NO !

**Review by Piras314 — CHANGES_REQUESTED** [review]

**Review by Piras314 — APPROVED** [review]

**Comment by whateverusername0** [comment]
nvm checks are related to rat species :trollface:

**Pull request merged**